index.tsx

```diff
@@ -0,0 +1,353 @@
+import React, {PropsWithChildren, useCallback, useMemo, useRef} from 'react';
```

Comment:
Please add rule to add space after "{" and before "}". That'll be more readable imo

Reply:
I didn't remember well, but I think the rule you want has a conflict with some other spacing rules (you can see it [here](https://github.com/oppia/oppia/issues/8128#issue-532742102)). I personally preferred the format you said but because of the conflict I couldn't resolve the conflict in my own projects too and it is still an open issue for Eslint.

<img width="832" alt="image" src="https://github.com/kesha-antonov/react-native-zoom-reanimated/assets/13766432/9ef89fec-bef8-4e70-9230-1a526d736e09">

Reply:
fixed